network-api/networkapi/templates/foundation-base.html

```diff
@@ -72,6 +72,34 @@
         </div>
       </div>
     </div>
+    {% if showHeroBanner %}
```

Comment:
you can change this if to use blocks.

Like this:

```
{% block hero_banner %}{% endblock %}
```

You can also fill in that block with a default banner unrelated to fellows, but I don't think we need to in our case.

Then in fellows_home.html somewhere you can do:

```
{% block hero_banner %}
    <div id="hero">
      <div class="container">
        <div class="row">
          <div class="col-md-12 col-lg-6 py-5">
            <div class="mb-4">
              <h1 class="h1-white">Fellowships</h1>
            </div>
            <p class="lead-black">Our Fellowships empower bright minds in the fields of public policy, science and journalism.</p>
          </div>
        </div>
      </div>
    </div>
{% endblock %}
```

and then remove the `, { 'showHeroBanner': True }` in fellows/views.py

Reply:
I think. I've  not tested this...

Reply:
Sounds good. Will fix!